test/isPair.js

```diff
@@ -0,0 +1,31 @@
+import RA from '../src/index';
+import eq from './shared/eq';
+
+describe('isPair', function() {
```

Comment:
Can we reword the `it` sections so that they are more consistent with ramda tests (all sections should start with test) ?

Example: https://github.com/char0n/ramda-adjunct/blob/40d74e6a2f1e11769e065ad012a4150989e458d8/test/isInteger.js#L8